docs/src/components/home/ProductsSwitcher.tsx

```diff
@@ -111,21 +111,21 @@ const ProductsSwitcher = ({
           MUI X
         </Box>
       }
-      description="Advanced and powerful components for complex use-cases."
+      description="Advanced components for complex use cases."
       href={ROUTES.productAdvanced}
     />,
     <ProductItem
       aria-label="Go to templates page"
       icon={<IconImage name="product-templates" />}
       name="Templates"
-      description="Fully built, out-of-the-box, templates for your application."
+      description="Battle-tested user interfaces, ready for production."
```

Comment:
```suggestion
      description="Battle-tested user interfaces, ready for production."
```
They're a starting point, not ready to ship.

Reply:
How about something like this?

```suggestion
      description="Battle-tested user interfaces to jumpstart your next project."
```

Reply:
```suggestion
      description="Professionally designed UI layouts to jumpstart your next project."
```
?